frontend_app_thread/src/css/index.styl

```diff
@@ -25,10 +25,11 @@
       width 100%
       &__content
         width 100%
+        height calc(100% - 175px)
       &__texteditor
         flex 0 0 auto
 
-@media (max-width: max-xs)
+@media (min-width: min-sm) and (max-width: max-lg)
   .thread
-    &__contentpage__content
-      height calc(100% - 185px) // wsContentGeneric:top wsContentGeneric__header:height - wsContentGeneric__option:height
+    .thread__contentpage__content
```

Comment:
This is right? Now you have
```
.thread
    .thread__contentpage__content
```

Reply:
yes, it is.
The class hierarchy of the app thread has some issues, here, I only tried to fix the CommentTextArea component height in responsive for the app thread